PTGLgraphComputation/src/proteingraphs/ComplexGraph.java

```diff
@@ -187,31 +231,43 @@ public ComplexGraph(String pdbid, List<Chain> chains, List<MolContactInfo> resCo
     
     /**
      * Creates the vertices, fills corresponding maps and fills amino acid sequence.
-     * @param chains All chains of this complex graph
+     * @param chains All chains of this complex graph. Hint: these chains are not the same objects as the ones that are parsed
```

Comment:
I do not understand the hint. The javadoc line should explain the parameter chains of this function. Do you want to say that createVertices expects something else then the parsed chains? Or does the hint refer to the actual function call? In the latter case, move the comment there.

Reply:
No it is about the way the function is used and not the function itself, so I removed it.